PCP_Chapter4_Exercises.cpp

```diff
@@ -14,51 +14,69 @@
 // Questions:
 //  How can I call a class that is in a different source file to the main function? 
 
-#include "stdafx.h"
+// use "using namespace blabla" sparingly - in general it is better to type out
+// the whole namespace to avoid ambiguity, and make the source clear.
+
+#include <iostream>
 using namespace std;
 
-// Called ex_42 for the exercise problem numbered 4.2 in the book. Holds functions that return a letter block (So far Hello) 
-class ex_42 {
-
-	public:
-		string Letter_H()
```

Comment:
Your audience are other software engineers - make reading your code as easy as possible. Legibility trumps most things, so keep that in mind next time you write functions. That said, there are of course cases when performant and concise code would win out, but this is not the case.